core/src/main/scala/com/emaginalabs/kuronometer/core/model/BuildExecutionReport.scala

```diff
@@ -0,0 +1,26 @@
+package com.emaginalabs.kuronometer.core.model
+
+case class ProjectInformation(name: Option[String], description: Option[String])
+
+case class ProjectVersion(group: Option[String], version: String)
+
+case class ProjectDefinition(info: ProjectInformation, version: ProjectVersion)
```

Comment:
The project definition itself has to be optional due to the anonymous report feature. You can keep the group as optional if you want :)